packages/react-scripts/scripts/eject.js

```diff
@@ -61,8 +65,10 @@ prompt(
   fs.mkdirSync(path.join(appPath, 'config', 'jest'));
   fs.mkdirSync(path.join(appPath, 'scripts'));
 
+  console.log();
+  console.log(cyan('Copying files...'));
   files.forEach(function(file) {
-    console.log('Copying ' + file + ' to ' + appPath);
+    console.log(cyan('  Copying ') + file+ ' to ' + appPath);
```

Comment:
Looking at the output, it prints the same giant appPath on every single line. 

Suggestion: could we put this path in the header instead?

```
Copying files to /Users/.../tempApp ...
  Copying .babelrc
  Copying .eslintrc
```